src/lib/components/UrlChooser/index.js

```diff
@@ -0,0 +1,185 @@
+import {html} from "lit-element";
+import {store} from "../../store";
+import {BaseElement} from "../BaseElement";
+import {userRef, firebase} from "../../fb";
+
+/* eslint-disable require-jsdoc */
+class UrlChooser extends BaseElement {
+  static get properties() {
+    return {
+      href: {type: String},
```

Comment:
should we call this `this.url` instead? Seems like that's how we otherwise refer to it with local variables.

Reply:
Done